### DISCUSSION TITLE
Negative emission values in REopt Lite

### DISCUSSION THREAD

**Discussion by AriH2023** [NREL/REopt-API-Analysis#124] [1 - General REopt Questions] [answered]
Using the online REopt Lite tool, I am getting results with negative emission values. For instance, for the Business As Usual case CO2 emissions are shown as 125,211 tonnes, while for the Financial case they are shown as -321,593 tonnes. Negative values also show up for other pollutants and for year 1 values. I would expect emission values to be above zero (due to emissions from grid power), not below 0. This used to work as expected when I previously ran REopt Lite. Why are the values now negative?

**Comment by adfarth** [comment, marked as answer]
This can happen (most likely) in cases in which a renewable energy system is sized such that it produces more electricity than the site consumes (an example of an extreme case: https://reopt.nrel.gov/tool/results/ebaef22e-297e-45c8-a550-92d132e4d50b). 

There is an input under the "Renewable Energy and Emissions" accordion titled "Count electricity exported to the grid towards emissions offsets?" The default is "Yes" but you can change this to "No" if, for your purposes, you would not like to count grid exports as emissions offsets. 

If you would like the share the URL to your results page with negative emissions, I can investigate further as well.

**Reply by AriH2023** [reply]
Thank you, that resolved the negative emission values.
